### PR TITLE
Make --name argument parsing stricter

### PR DESCRIPTION
**Pull request by errordeveloper**
Hey @christopherhein, I managed to incorporate some of my improvements on top of #81.

### CODE REVIEW

**Comment by errordeveloper** [comment]
I've overlooked what `fs.MarkDeprecated` does, and it's really unnecessary anyway, anyone using `--cluster-name` will just see an error and  cobra prints usage info right away, so no need to make it all redundant for little benefit. I removed that and re-tested – all works nicely now. Please review once again so we get green tick and I'll merge. Perhaps we should cut another alpha after this.

**Comment by errordeveloper** [comment]
@christopherhein would you mind to review it once again? Thanks!

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -14,7 +14,16 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-func ClusterName() string {
+func ClusterName(a, b string) string {
```
Comment (stefanprodan) [inline pkg/utils/utils.go]: @errordeveloper could you add a comment describing the logic in here, why you return string empty when both values are not nil?

Comment (errordeveloper) [reply pkg/utils/utils.go]: 👍

**Comment by errordeveloper** [comment]
@stefanprodan I've documented the behaviour of this function – PTAL.

**Pull request merged**